stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/viewholder/SwipeViewHolder.kt

```diff
@@ -26,6 +26,11 @@ import kotlin.math.absoluteValue
 
 public abstract class SwipeViewHolder(itemView: View) : BaseChannelListItemViewHolder(itemView) {
 
+    /**
+     * Swipe threshold
+     */
+    private val swipeThreshold = SWIPE_THRESHOLD.dpToPxPrecise()
```

Comment:
Having it here will increase the memory used, because we are going to have one property per `SwipeViewHolder` instance